DMX.Portal.Web.Infrastructure.Provision/Program.cs

```diff
@@ -0,0 +1,7 @@
+﻿// --------------------------------------------------------------- 
+// Copyright (c) Microsoft Corporation. All rights reserved. 
+// ---------------------------------------------------------------
+
+using System;
+
+Console.WriteLine("Hello, World!");
```

Comment:
@Geetika Jain we still need to call the processing service to actually execute the provisioning process